src/deck-options.md

```diff
@@ -144,7 +144,7 @@ This option can be useful if you wish to study individual subdecks, while enforc
 ## New Cards
 
 The options here only affect new cards and
-[learning cards](studying.md#learningrelearning-cards). Once a card
+[learning cards](getting-started.md#card-states). Once a card
```

Comment:
Is this the right link to substitute? It doesn't have the same information as the moved/removed section.

Reply:
That info was moved to this very topic.